cpp/sophus/interp/spline/details/cubic_basis.h

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2011, Hauke Strasdat
```

Comment:
Moved and light refactored (more precise names) from https://github.com/farm-ng/farm-ng-core/blob/038a0c3f6ed6be19b95fa6cc2e3e0153b474c31d/cpp/sophus/lie/interp/spline.h